ampcrowd/amt/connection.py

```diff
@@ -61,14 +76,24 @@ def create_hit(hit_options):
         frame_height=options['frame_height'])
     conn = get_amt_connection(options['sandbox'])
 
-    create_response = conn.create_hit(
-        question=question,
-        title=options['title'],
-        description=options['description'],
-        reward=Price(amount=options['reward']),
-        duration=timedelta(minutes=options['duration']),
-        max_assignments=options['num_responses'],
-        approval_delay=0)
+    try:
+        create_response = conn.create_hit(
+            question=question,
+            title=options['title'],
+            description=options['description'],
+            reward=Price(amount=options['reward']),
+            duration=timedelta(minutes=options['duration']),
+            max_assignments=options['num_responses'],
+            approval_delay=0)
+    except MTurkRequestError:
```

Comment:
It was actually already imported here: https://github.com/EntilZha/ampcrowd/blob/better-logging/ampcrowd/amt/connection.py#L8